### ISSUE TITLE
Add .json output

### ISSUE THREAD

**Issue by tseemann** [tseemann/mlst#38]
Two ways to do allelles though:

Either    alleles: [ a,b,c,d,e ]
Or a => 13, b=>2, c=>1 etc

**Comment by tseemann** [comment]
Added `--json OUTFILE` option.

**Comment by Myskmadra** [comment]
Hi,

I really appreciate the json output option. I just have one comment/suggestion as I want to use the output in a workflow. I would be better to use consistent field names instead of being dependent on the data input. For example instead of

```json
{
  "MySample":
    {
      ...
    }
}
```

it would be nice to have 

```json
{
  "Sample": "MySample"
  "Values":
    {
      ...
    }
}
```

This makes it more predictable when working with the json output files.

**Comment by tseemann** [comment]
@Myskmadra what format would you expect when there are multiple samples though? an array or a hash?  eg. `mlst *.fa`

**Comment by Myskmadra** [comment]
I would expect an array if multiple samples are processed.

**Comment by tseemann** [comment]
@Myskmadra i've changed the JSON output to an array and released 2.19.0
Thank you!
